src/openrct2/ride/Vehicle.cpp

```diff
@@ -2682,15 +2682,12 @@ static void vehicle_update_waiting_to_depart(Vehicle* vehicle)
     }
 }
 
-#pragma pack(push, 1)
 struct rct_synchronised_vehicle
 {
     ride_id_t ride_id;
     uint8_t station_id;
     uint16_t vehicle_id;
 };
-assert_struct_size(rct_synchronised_vehicle, 4);
```

Comment:
Why removing instead of updating? Just wondering

Reply:
They were originally used to ensure we kept size compatibility with vanilla when interoping. We don't introp anymore and this struct is not used in the save file, or in networking so we can let the compiler adjust the struct however it likes.